hints_tests.py

```diff
@@ -23,7 +23,7 @@ def test_program(program_name: str):
     test_program("assert_lt_felt")
     test_program("assert_not_equal")
     test_program("reduce_and_nondet_bigint3")
-    test_program("is_zero")
+    # test_program("is_zero") # Custom Hint Error: NameError: name 'to_felt_or_relocatable' is not defined
```

Comment:
This should be uncommented, isn't the point of the PR to fix this test?

Reply:
Yes, must have overided it when updating the branch